lib/src/types/page_introduction.dart

```diff
@@ -0,0 +1,420 @@
+              );
+            }),
+      ],
+    );
+  }
+
+  bool _isLast(List<IntroductionPage> pages) =>
+      _currentPage.value == pages.length - 1;
+
+  bool get _isPrevious => _currentPage.value > 0;
+
+  bool _isNext(List<IntroductionPage> pages) =>
+      _currentPage.value < pages.length - 1;
+}
+
+class ExplainerPage extends StatelessWidget {
+  const ExplainerPage({
+    required this.page,
+    required this.options,
+    required this.index,
+    required this.controller,
+    this.onTap,
+    Key? key,
+  }) : super(key: key);
+
+  final IntroductionPage page;
+  final IntroductionOptions options;
+  final PageController controller;
+  final int index;
+  final VoidCallback? onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    var theme = Theme.of(context);
+    return Background(
+      backgroundImage: page.backgroundImage,
+      child: SafeArea(
+        child: Column(
+          children: [
+            const SizedBox(
+              height: 64,
+            ),
+            Expanded(
+              child: IntroductionPageContent(
+                onTap: () {
+                  if (options.tapEnabled) {
+                    onTap?.call();
+                  }
+                },
+                title: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 32),
+                  child: DefaultTextStyle(
+                    style: theme.textTheme.displayMedium!,
+                    child: page.title ?? Text('introduction.$index.title'),
+                  ),
+                ),
+                text: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 32),
+                  child: DefaultTextStyle(
+                    style: theme.textTheme.bodyLarge!,
+                    child: page.text ?? Text('introduction.$index.description'),
+                  ),
+                ),
+                graphic: Expanded(
+                  child: Padding(
+                    padding: const EdgeInsets.all(32),
+                    child: page.graphic,
+                  ),
+                ),
+                options: options,
+              ),
+            ),
+            const SizedBox(
+              height: 144,
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class IntroductionButtons extends StatelessWidget {
+  const IntroductionButtons({
+    required this.options,
+    required this.controller,
+    required this.next,
+    required this.last,
+    required this.previous,
+    required this.onFinish,
+    required this.onNext,
+    required this.onPrevious,
+    Key? key,
+  }) : super(key: key);
+
+  final IntroductionOptions options;
+  final PageController controller;
+  final VoidCallback? onFinish;
+  final VoidCallback? onNext;
+  final VoidCallback? onPrevious;
+
+  final bool previous;
+  final bool next;
+  final bool last;
+
+  void _previous() {
+    controller.previousPage(
+      duration: const Duration(milliseconds: 100),
```

Comment:
previousPage = 100 milliseconds, nexPage = 200 en naar page vanaf indicator is 500.
Waarom?